sample/lit-todo/rollup.config.js

```diff
@@ -0,0 +1,10 @@
+import {nodeResolve} from '@rollup/plugin-node-resolve';
+
+export default {
+  input: 'main.js',
+  output: {
+    dir: 'output',
+    format: 'cjs',
```

Comment:
output should be esm

Reply:
I'm not actually sure what difference this makes given there are no imports, but done.